doc/changelog.md

```diff
@@ -0,0 +1,925 @@
+# Changelog
+
+Listed here are the changes between each release of SmartSim,
+SmartRedis and SmartDashboard.
+
+Jump to {ref}`SmartRedis changelog<smartredis-changelog>` or {ref}`SmartDashboard Changelog<smartdashboard-changelog>`
+
```

Comment:
Maybe this should be a bulleted list? Something like
```
Jump to:
- SmartRedis changelog
- SmartDashboard changelog
```

Let me know what you think!